app/components/InstanceManagerModal/Settings/JavaManagerCard.js

```diff
@@ -0,0 +1,273 @@
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
+import { Form, Input, Button, message, Switch, Tooltip } from 'antd';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faQuestionCircle } from '@fortawesome/free-solid-svg-icons';
+import path from 'path';
+import { promisify } from 'util';
+import fs, { stat } from 'fs';
+import { exec } from 'child_process';
+import log from 'electron-log';
+import _ from 'lodash';
+import Card from '../../Common/Card/Card';
+import styles from './JavaManagerCard.scss';
+import { PACKS_PATH, DEFAULT_ARGS } from '../../../constants';
+import { setJavaArgs } from '../../../actions/settings';
+import { findJavaHome } from '../../../utils/javaHelpers';
+import MemorySlider from './MemorySlider';
+import JavaArgInput from './JavaArgInput';
+
+function JavaManagerCard(props) {
+  const [is64bit, setIs64bit] = useState(true);
+  const [overrideArgs, setOverrideArgsInput] = useState(null);
+  const [javaArgsSwitchState, setJavaArgsSwitchState] = useState(false);
+  const [javaMemorySwitchState, setJavaMemorySwitchState] = useState(false);
+  const [overrideJavaMemory, setOverrideJavaMemory] = useState(null);
+  const [memory, setMemory] = useState(null);
+
+  const updateJavaArguments = javaArguments => {
+    setOverrideArgsInput(javaArguments);
+  };
+
+  // resetArgs the global arguments to the default one
+  const resetArgs = async () => {
+    updateJavaArguments(DEFAULT_ARGS);
+    const config = JSON.parse(
+      await promisify(fs.readFile)(
+        path.join(PACKS_PATH, props.instanceName, 'config.json')
+      )
+    );
+    const modifiedConfig = JSON.stringify({
+      ...config,
+      overrideArgs: DEFAULT_ARGS
+    });
+    await promisify(fs.writeFile)(
+      path.join(PACKS_PATH, props.instanceName, 'config.json'),
+      modifiedConfig
+    );
+  };
+
+  const checkJavaArch = async () => {
```

Comment:
This is a duplicate of https://github.com/gorilla-devs/GDLauncher/blob/master/app/components/Settings/components/JavaManager/JavaManager.js#L25